server/routes/choosePrisonerController.ts

```diff
@@ -6,7 +6,8 @@ export default class ChoosePrisonerController {
 
   public view(): RequestHandler {
     return async (req, res) => {
-      const incomingMovements = await this.incomingMovementsService.getIncomingMovements('MDI')
+      const { activeCaseLoadId } = res.locals.user
+      const incomingMovements = await this.incomingMovementsService.groupByMoveType(activeCaseLoadId)
```

Comment:
`getMovesGroupedByType` ??

Reply:
updated function name to suggested `getMovesForToday`